### ISSUE TITLE
page method

### ISSUE THREAD

**Issue by jenstornell** [AugustMiller/kirby-architect#4]
Something like this:

``` php
echo $page->architect($args);
```

The benefit is that this page contains an intended template and the page blueprint can be used.

**Comment by AugustMiller** [comment]
I think I'm going to add the following to the Readme, rather than including it in the repository itself:

``` php
kirby()->set('page::method', 'architect', function($page) {
  return Architect::blueprint($page->intendedTemplate());
});
```

This way, developers can add the functionality easily, without the method name colliding with any of their fields.

One major caveat: I [opened a thread](https://forum.getkirby.com/t/using-the-registry-from-config-php/5198) about the trouble I had making page method extensions within `config.php`. Whoever is implementing the plugin has to drop these into a new plugin file, not their config file…

**Comment by jenstornell** [comment]
Go for a plugin approach. At least I would. Maybe even make an example plugin with all the available field  and page methods.

**Comment by AugustMiller** [comment]
Implemented in the readme in f0bebd2de5d91ab358992f67591e71badcbf5a35.
